unique_sdk/unique_sdk/_http_client.py

```diff
@@ -16,21 +28,59 @@ def new_default_http_client(*args, **kwargs) -> "HTTPClient":
     return impl(*args, **kwargs)
 
 
+def new_http_client_async_fallback(*args, **kwargs) -> "HTTPClient":
+    if aiohttp:
+        impl = AIOHTTPClient
+    elif httpx:
+        impl = HTTPXClient
+    return impl(*args, **kwargs)
+
+
 class HTTPClient(object):
     name: ClassVar[str]
 
-    def __init__(self):
+    def __init__(
+        self,
+        async_fallback_client: Optional["HTTPClient"] = None,
```

Comment:
As long as httpx or aiohttp are installed as dependency, there will always be a fallback client. If not, it will only fail when you make an async request but has only requests lib installed.